server/mem_agent/tools.py

```
@@ -0,0 +1,359 @@
+    """
+    Delete a file in the memory.
+
+    Args:
+        file_path: The path to the file.
+
+    Returns:
+        True if the file was deleted successfully, False otherwise.
+    """
+    try:
+        os.remove(file_path)
+        return True
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)